analyticConfigurations/computed_metric-consul.json

```diff
@@ -0,0 +1,28 @@
+{
+  "analyticConfiguration": {
+    "metrics": [
+      {
+        "match": "consul.raft.commitTime.max",
+        "properties": {
+          "expression": "metric['consul.raft.commitTime'].hmax",
```

Comment:
This is the historical maximum, the highest value seen in the history of this metric... not sure if that is what you actually want here?

Reply:
should just be `.max` I believe